tests/beacon/types/test_block.py

```diff
@@ -40,6 +27,20 @@ def test_update_attestations(sample_attestation_record_params, sample_block_para
     assert block2.num_attestations == 1
 
 
-def test_hash(sample_block_params):
-    block = BaseBeaconBlock(**sample_block_params)
+def test_hash(sample_beacon_block_params):
+    block = BaseBeaconBlock(**sample_beacon_block_params)
     assert block.hash == blake(rlp.encode(block))
+
+
+def test_parent_hash(sample_beacon_block_params):
+    block = BaseBeaconBlock(**sample_beacon_block_params)
```

Comment:
*nitpick*.  You can inline the `copy` here to prevent overwriting the `block` variable. 

```python
block = BaseBeaconBlock(**sample_beacon_block_params).copy(ancestor_hashes=())
```